src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let arr = [];
```

Comment:
Use some more meaningful name, so that it's clear what this array is for without need to look what you push there somewhere lower in the code